src/visual.ts

```diff
@@ -296,7 +296,7 @@ module powerbi.extensibility.visual {
                     }
                     let streamDataPoint: StreamDataPoint = {
                         x: dataPointValueIndex,
-                        y: isNaN(y)
+                        y: isNaN(y) || y === null
```

Comment:
Please create a method that checks the value since you have the same conditions at two places.